private/bufpkg/buflock/buflock.go

```diff
@@ -122,6 +130,7 @@ func DependencyForExternalConfigDependencyV1Beta1(dep ExternalConfigDependencyV1
 		Owner:      dep.Owner,
 		Repository: dep.Repository,
 		Commit:     dep.Commit,
+		Digest:     "", // digests in v1Beta1 are not valid v1 digests
```

Comment:
A bit confused, why?

Reply:
v1Beta1 never understood a digest we support. No sense in trying to support buf beta when a default ignore is just the same.